.prettierrc.json

```diff
@@ -2,5 +2,6 @@
   "semi": true,
   "singleQuote": true,
   "trailingComma": "all",
-  "printWidth": 100
+  "printWidth": 100,
+  "arrowParens": "avoid"
```

Comment:
this line should not be needed, as it's the default setting in prettier2

Reply:
The defaults was changed to `always`, so not to change code I've changed the settings. https://prettier.io/docs/en/options.html#arrow-function-parentheses